packages/fgtclb/academic-projects/Classes/Domain/Model/Dto/ProjectDemand.php

```diff
@@ -55,14 +55,18 @@ public function getShowSelected(): bool
         return $this->showSelected;
     }
 
-    public function setHideCompletedProjects(bool $hideCompletedProjects): void
```

Comment:
Removing public facing methonds, special on DTO/Models is technically a breaking change.

That means, the commit and pull-request needs to get:

* Breaking prefix `[!!!]`
* Migration path / Breaking statement in extension `UPGRADE.md` file.